### PR TITLE
[suitespare] updated the pinning to the latest 5.4 for Windows.

### PR DESCRIPTION
**Pull request by seanyen**
Checklist
* [ ] Used a [fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by seanyen** [comment]
Now the 1suitesparse` for Windows has been updated to `5.4` with the latest `blas` convention. I am suggesting to update the pinning.
https://github.com/conda-forge/suitesparse-feedstock/pull/65

**Comment by seanyen** [comment]
@conda-forge/core This is ready for review and merge. Thanks!

**Comment by CJ-Wright** [comment]
This most likely needs a migration

**Comment by seanyen** [comment]
> This needs a migration (most likely)

@CJ-Wright What does that mean? How do I make it happen?

**Comment by CJ-Wright** [comment]
Put a copy of [this file](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/master/recipe/migrations/example.exyaml) in the `migrations` folder of this repo.
Then modify the file to match the changes you made here:
```yaml
suitesparse:
  - 5.6                # [not win]
  - 5.4                # [win]
```

**Comment by seanyen** [comment]
@CJ-Wright Thanks for the tip! Hope I get the migration description right...

**Comment by isuruf** [comment]
Btw, how many existing packages use suitesparse on windows? I assume one or two

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -589,7 +589,7 @@ sqlite:
   - 3
 suitesparse:
   - 5.6                # [not win]
-  - 5.1                # [win]
+  - 5.4                # [win]
```
Comment (CJ-Wright) [inline recipe/conda_build_config.yaml:592]: you need to remove this change this will go in once the migration has finished.

**Comment by seanyen** [comment]
> Btw, how many existing packages use suitesparse on windows? I assume one or two

@isuruf Frankly I don't know. Any commands to quickly check it? And it seems to me that there are two ways: Pinning a version in `conda_build_config.yaml` or adding a migrator. What's the consideration which to use?

**Comment by isuruf** [comment]
Looks like it is only cvxopt. (Inspected https://conda.anaconda.org/conda-forge/win-64/repodata.json)
So, if you can revert back to the original commit, I'll merge.

There's no point doing a migration for 1 package.

**Comment by seanyen** [comment]
@isuruf Done. I removed the migrator file.

**Pull request merged**